src/promptflow/promptflow/_utils/load_data.py

```diff
@@ -79,7 +79,7 @@ def _handle_dir(dir_path: str, max_rows_count: int, logger: logging.Logger = Non
     while len(target_dir) > 0:
         files, dirs = _bfs_dir(target_dir)
         for file in files:
-            current_df = _pd_read_file(file, logger=logger)
+            current_df = _pd_read_file(file, logger=logger, max_rows_count=max_rows_count)
             df = pd.concat([df, current_df])
```

Comment:
Please also add a check here to see if it exceeds the `max_rows_count`.

Reply:
Fixed. Thanks.